cmd/gazelle/update-repos.go

```diff
@@ -136,10 +137,19 @@ func updateRepos(wd string, args []string) (err error) {
 	}
 	uc := getUpdateReposConfig(c)
 
+	moduleToApparentName, err := module.ExtractModuleToApparentNameMapping(c.RepoRoot)
+	if err != nil {
+		return err
+	}
+
 	kinds := make(map[string]rule.KindInfo)
 	loads := []rule.LoadInfo{}
 	for _, lang := range languages {
-		loads = append(loads, lang.Loads()...)
```

Comment:
Can we simply replace all repos in the `loads` with their apparent names here automatically? Is there any case when an extension don't want to replace repo names with apparent names?

Reply:
I would have liked to do so, but I don't think it's possible without extra information: We somehow need to know that the WORKSPACE repo `io_bazel_rules_go` is equivalent to the `rules_go` module before we can figure out the apparent repo name for that.

Instead of `ApparentLoads`, we could introduce a `ModuleName` function that returns the module name for a given legacy WORKSPACE name. The resulting API would be simpler, but I don't like that the "interface" would still be WORKSPACE names, which are slated to go away.